packages/server/package.json

```
@@ -18,8 +18,11 @@
   "preconstruct": {
     "entrypoints": [
       "index.ts",
-      "adapters/next.ts",
       "adapters/express.ts",
+      "adapters/next.ts",
+      "adapters/node-http/index.ts",
```

maybe call it just `http` which would quite nicely reflect `require('http')`? 🤔 

```suggestion
      "adapters/http/index.ts",
````